app/configs/mtl/modules.conf

```diff
@@ -20,3 +20,4 @@ CONFIG_COMP_CROSSOVER=m
 CONFIG_COMP_MULTIBAND_DRC=m
 CONFIG_COMP_GOOGLE_CTC_AUDIO_PROCESSING=m
 CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=m
+CONFIG_COMP_TESTER=m
```

Comment:
It would be nice to have the common non-intel changes in one commit and these Intel board changes in a separate.

Reply:
@marcinszkudlinski Not a blocking thing, but this board file change is still in the same commit as the tester module implementation. So a comment would be nice if you just prefer not to do this.